src/components/app-text-area/AppTextArea.tsx

```diff
@@ -18,11 +18,19 @@ const AppTextArea: FC<AppTextAreaProps> = ({
   minRows = 4,
   maxRows = 4,
   maxLength,
+  title,
   value,
+  sx,
   ...props
 }) => {
+  const titleEl = title && (
+    <Typography sx={styles.title} variant='body2'>
```

Comment:
i think it would be good to use enum of VariantEnum for variant property

Reply:
fixed